python/spec/fixtures/poetry_locks/python_lower_bound.toml

```diff
@@ -0,0 +1,149 @@
+# This file is automatically @generated by Poetry 1.5.1 and should not be changed by hand.
```

Comment:
```suggestion
# This file is automatically @generated by Poetry 1.6.1 and should not be changed by hand.
```

Since I just merged a version bump which had no changes to the lockfile format:
* https://github.com/dependabot/dependabot-core/pull/7866